File: Iniciante/1046 - Tempo de Jogo.py
"""
Autor: Pierre Vieira
Data da submissão: 24/01/2019 20:42:12
"""
linha = input().split(" ")
a, b = linha
hora_inicial = int(a)
hora_final = int(b)
if hora_final > hora_inicial:
    duracao = hora_final - hora_inicial
elif hora_final == hora_inicial:
    duracao = 24
else:
    cont = 0
    for c in range(hora_inicial, 24):
        cont+=1
    duracao = cont + hora_final
print("O JOGO DUROU {} HORA(S)".format(duracao))

File: C#/Iniciante/1046 - Tempo de Jogo.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 14/05/2020 10:22:40
*/
using System;
using System.Globalization;

namespace URI {
    class URI {
        static void Main(string[] args) {
            string[] linha = Console.ReadLine().Split(' ');
            int d1, d2;
            d1 = int.Parse(linha[0]);
            d2 = int.Parse(linha[1]);
            if (d1 == d2) {
                Console.WriteLine("O JOGO DUROU 24 HORA(S)");
            }
            else if (d1 < d2)
            {
                Console.WriteLine($"O JOGO DUROU {d2 - d1} HORA(S)");
            }
            else
            {
                Console.WriteLine($"O JOGO DUROU {24 - d1 + d2} HORA(S)");
            }
        }
    }
}
